### DISCUSSION TITLE
Progress notification for normal actions across all apps

### DISCUSSION THREAD

**Discussion by ghost** [FossifyOrg/General-Discussion#147] [Ideas]
It will be more convenient for users to have a progress notification across all apps for tasks like copy, cut and paste, import, export, compressing etc. Sometimes progress is still going on in background but confuses user that it either stopped or have some error. Originally suggested by @tom93

**Comment by Aga-C** [comment]
It's already reported here: https://github.com/FossifyOrg/General-Discussion/issues/120.
